client/src/components/qa/AddAnswer.jsx

```diff
@@ -0,0 +1,101 @@
+/* eslint-disable react/jsx-one-expression-per-line */
+/* eslint-disable jsx-a11y/no-static-element-interactions */
+import React, { useState } from 'react';
+import axios from 'axios';
+
+function AddAnswer({
+  id, name, body, setAnsOpen, getQuestions,
+}) {
+  const [images, setImages] = useState([]);
+  const [answerInput, setAnswerInput] = useState('');
+  const [answerName, setAnswerName] = useState('');
+  const [answerEmail, setAnswerEmail] = useState('');
+
+  const close = () => {
+    setAnsOpen(false);
+  };
+  const handleAddAnswer = (e) => {
+    e.preventDefault();
+    axios({
+      method: 'post',
+      url: `/qa/questions/${id}/answers?${id}`,
+      data: {
+        body: answerInput,
+        name: answerName,
+        email: answerEmail,
+        photos: images,
+      },
+    })
+      .then(() => {
+        console.log('success');
+        close();
+        getQuestions();
+      })
+      .catch((err) => {
+        console.log(err);
+      });
+  };
+  const handleImg = (e) => {
+    setImages([...images, URL.createObjectURL(e.target.files[0])]);
+  };
+
+  const content = new Array(1).fill(
```

Comment:
It's a bit unclear why are you creating an array that holds the content. Does it serve some other purpose aside from for rendering the q&a modal? If it's used only in the q&a modal, I think you can just put all this code in the q&a modal div. On the other hand, I can see that it might look too bulky, not sure what is the best approach in that case though.